src/cryptonote_core/service_node_list.cpp

```diff
@@ -406,7 +406,6 @@ namespace service_nodes
         info.decommission_count++;
 
         info.proof.timestamp = 0;
-        info.proof.votes.fill(true);
```

Comment:
We need to keep this here as well, I think, for the reasons we discussed earlier (without it you can't get recommissioned because you'll always fail the missed vote check).

Reply:
In check_service_node we check if (!is_decommissioned()) though for checkpoint obligations which should handle that?

Reply:
Oh, that's right.